### PR TITLE
Add calendar panel to show schedule 

### PR DESCRIPTION
**Pull request by qiujiaaa**
still trying to get the agenda to show

### CODE REVIEW

File: src/main/java/seedu/address/ui/CalendarPanel.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.schedule.Schedule;
+import seedu.address.ui.cards.ScheduleCard;
+
+/**
+ * Panel containing the calendar.
+ */
+public class CalendarPanel {
+
+    // private static final String FXML = "ScheduleListPanel.fxml";
```
Comment (uberSaiyan) [inline src/main/java/seedu/address/ui/CalendarPanel.java]: Is this gonna be added in the future?

Comment (EugeneTeu) [reply src/main/java/seedu/address/ui/CalendarPanel.java]: she changed to Calendar panel

File: src/main/java/seedu/address/ui/CalendarPanel.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        super(FXML);
+
+        this.scheduleList = scheduleList;
+        this.orderList = orderList;
+
+        agenda = new Agenda();
+        populateAgenda();
+
+        scheduleList.addListener((ListChangeListener<Schedule>) change -> populateAgenda());
+
```
Comment (EugeneTeu) [inline src/main/java/seedu/address/ui/CalendarPanel.java]: might want to add a comment here to indicate the set up of the listener

File: src/main/resources/view/CalendarPanel.fxml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.layout.*?>
+
+<StackPane styleClass="stack-pane" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
+    <VBox xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1" fx:id="calendarBox">
+    </VBox>
```
Comment (EugeneTeu) [inline src/main/resources/view/CalendarPanel.fxml]: might want to give the V-box some paddings around it

File: src/main/java/seedu/address/ui/CalendarPanel.java [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+     */
+    private LocalDateTime scheduleToLocalDateTime(Schedule schedule) {
+        Calendar calendar = schedule.getCalendar();
+        int year = calendar.get(Calendar.YEAR);
+        // offset to 1-based
+        int month = calendar.get(Calendar.MONTH) + 1;
+        int date = calendar.get(Calendar.DAY_OF_MONTH);
+        int hour = calendar.get(Calendar.HOUR_OF_DAY);
+        int minute = calendar.get(Calendar.MINUTE);
+        return LocalDateTime.of(year, month, date, hour, minute);
```
Comment (EugeneTeu) [inline src/main/java/seedu/address/ui/CalendarPanel.java]: might affect my statsUtil in the future. we should take note

**Pull request merged**